### ISSUE TITLE
Corrupt libsvm version packaged

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/contrib#36]
https://trac.macports.org/ticket/37862
We should upgrade. Probably related to failing SVM related tests on some of our builds.

**Comment by hroest** [comment]
it seems to me we already discussed exactly this issue before: https://github.com/OpenMS/OpenMS/issues/521

**Comment by hroest** [comment]
yes, I think we decided to use 3.12 -- we should probably add a comment to the contrib code to read those tickets before we change the version number

**Comment by hroest** [comment]
libsvm is now at version 3.23 and Ubuntu 18.04 packages 3.21 (see https://launchpad.net/ubuntu/bionic/+package/libsvm-dev) so we should start to look at our code and whether we can fix it for a newer version of libsvm
